btr-api/src/btr_api/resources/submission.py

```diff
@@ -175,3 +178,71 @@ def create_register():
         return exception_response(aex)
     except Exception as exception:  # noqa: B902
         return exception_response(exception)
+
+
+@bp.route('/<sub_id>', methods=('PUT',))
+@cross_origin(origin='*')
+@jwt.requires_auth
+def update_submission(sub_id: int):
+    """
+    Update an existing submission.
+    Any fields that are not in the payload will be ignored.
+    To remove a field the payload explicitly pass null
+
+    Returns:
+        A tuple containing the response JSON and the HTTP status code.
+    """
+    try:
+        user = UserModel.get_or_create_user_by_jwt(g.jwt_oidc_token_info)
+        account_id = request.headers.get('Account-Id', None)
+        submission = SubmissionModel.find_by_id(sub_id)
```

Comment:
Do we have submission ID in the UI already, or do we need ticket/update to get that information ?

Reply:
We do have it in the UI already